doc/integrations/images-api.md

```diff
@@ -0,0 +1,33 @@
+**What is Images api?**
```

Comment:
This file needs to be linked from the top-level page:
https://github.com/Seagate/cortx/tree/main/doc/integrations